scripts/release.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env sh
+set -e
+PATH=$PATH:$PWD/bin
+GOPATH=$PWD
+
+SRC=${PWD}/src/github.com/SUSE/cf-plugin-backup
+TAG=$(git -C ${SRC} describe --tags --long | awk -F - '{ print $1 }' )
+NAME=$(basename $(git -C ${SRC} config --get remote.origin.url) .git)
+
+# Assemble configuration data for `out` to git-release resource.
+echo > release/name ${TAG}
+echo > release/tag  ${TAG}
+echo > release/body "${NAME}: Release ${TAG}"
+
+# Build the tarballs for the release
+make -C ${SRC} build dist
+
+# Show configuration in log
+echo Release information:
+for path in release/*
+do
+    printf "%s: " "${path#**/}"
+    cat $path
+done
+
+ #Unpack the tarballs into the release.
+mkdir release/assets
+for path in ${SRC}/*.tgz
+do
+    echo Processing ${path} ...
+
+    rm -rf tmp
+    mkdir  tmp
+    ( cd tmp ; tar xfz $path )
```

Comment:
```bash
tar xfz "${path}" \
    -C release/assets/ \
    --strip-components=1 \
    --overwrite \
    --transform="s@/cf-plugin-backup@/${path%%.tgz}@"
```

Reply:
huh. that looks complicated. will have a look

Reply:
Done with commit 26ec7b3de6826d8677420fe18dc7d72b1f188c88
Note, one fix from my side:
- `path` comes with leading directories from `SRC`.
- Not removing them causes `tar` to attempt to write to `release/assets/foo/bar/cf-...` which silently skips the failing extraction.
- I added `basename` back around the `${path%%.tgz}` to drop these leading dirs.
- With that change the `tar` works.